src/beast/beast/nudb/common.h

```diff
@@ -17,10 +17,9 @@
 */
 //==============================================================================
 
-#ifndef BEAST_NUDB_ERROR_H_INCLUDED
-#define BEAST_NUDB_ERROR_H_INCLUDED
+#ifndef BEAST_NUDB_COMMON_H_INCLUDED
+#define BEAST_NUDB_COMMON_H_INCLUDED
 
-#include <beast/nudb/detail/config.h>
 #include <beast/utility/noexcept.h>
```

Comment:
The problem is that when code changes, there's no way to know if some includes are no longer needed :-(